style.css

```diff
@@ -0,0 +1,59 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #f5f5f5;
+  font-family: Poppins, sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+#app {
+  background: #fff;
+  width: 100vw;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 50px;
+  padding: 20px;
+}
+
+.header h2 {
+  color: #6070ff;
+}
+
+.showcase {
+  background-image: url("./img/header-shapes-mobile.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 75vh;
+  padding: 20px;
+}
```

Comment:
- [ ] As pointed by the requirement screenshotted below, please use flexbox here, while the end result is gonna look the same it is a requirement because learning how to use it is one of the learning objectives for this milestone.
![image](https://user-images.githubusercontent.com/73140461/148433636-39312f28-f67a-4731-a879-3c42455b75fe.png)